src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  top: -97px;
+  left: -601px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu ul {
+  width: 100%;
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (like `ul` here). Instead, style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.